services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchFailuresSpec.groovy

```diff
@@ -70,78 +79,120 @@ class SwitchFailuresSpec extends HealthCheckSpecification {
         }
     }
 
-    @Ignore("This is a known Kilda limitation and feature is not implemented yet.")
+    @Ignore("Not ready yet")
+    //expected to work only via v2 API
     def "System can handle situation when switch reconnects while flow is being created"() {
-        given: "Source and destination switches"
+
+        when: "Start creating a flow between switches and lose connection to src before rules are set"
         def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
+        def flow = flowHelperV2.randomFlow(srcSwitch, dstSwitch)
+        northboundV2.addFlow(flow)
+        sleep(50)
+        lockKeeper.knockoutSwitch(srcSwitch)
 
-        when: "Start creating a flow between these switches"
-        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
-        def addFlow = new Thread({ northbound.addFlow(flow) })
-        addFlow.start()
+        then: "Flows is 'In progress' retrying to install rules while switch is still marked as ACTIVATED"
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northbound.getSwitch(srcSwitch.dpId).state == SwitchChangeType.DEACTIVATED
```

Comment:
looks like you forgot to update the step description
you wrote `Flows is 'In progress'` and `switch is still marked as ACTIVATED`
in fact you check:
```
assert northbound.getSwitch(srcSwitch.dpId).state == SwitchChangeType.DEACTIVATED
 assert northbound.getFlowStatus(flow.flowId).status != FlowState.IN_PROGRESS
```